### PR TITLE
i18n(id): Update sidebar.mdx translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description
- Update ID translation guides for sidebar.mdx file

### CODE REVIEW

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 1]
```diff
@@ -51,31 +49,85 @@ Pelajari lebih lanjut tentang sidebar yang dihasilkan secara otomatis di bagian
...
+```
+
+### Tautan lainnya
+
+Tambahkan tautan ke halaman eksternal atau non-dokumen menggunakan objek dengan properti `label` dan `link`.
+
+```js "label:" "link:"
+starlight({
+	sidebar: [
+		// A link to a non-docs page on this site.
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:117]: This comment can be translated.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 2]
```diff
@@ -51,31 +49,85 @@ Pelajari lebih lanjut tentang sidebar yang dihasilkan secara otomatis di bagian
...
+		{ label: 'Toko Meteor', link: '/shop/' },
+		// An external link to the NASA website.
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:119]: And this one too.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 3]
```diff
@@ -87,22 +139,22 @@ Grup dapat berisi baik tautan maupun sub-grup lainnya.
...
 
-```js
+```js /^\s*(label:|items:)/
 starlight({
 	sidebar: [
-		// Grup tautan dengan label "Panduan".
+		// Sekelompok tautan yang diberi label "Constellations".
 		{
-			label: 'Panduan',
+			label: 'Constellations',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:147]: This label can be safely translated, would it make sense in this language?

If yes, the example below should also be updated too.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 4]
```diff
@@ -87,22 +139,22 @@ Grup dapat berisi baik tautan maupun sub-grup lainnya.
...
 			items: [
-				{ label: 'Komponen', link: '/guides/components/' },
-				{ label: 'Internasionalisasi (i18n)', link: '/guides/i18n/' },
-				// Grup tautan bertingkat
+				'constellations/carina',
+				'constellations/centaurus',
+				// Sekelompok tautan bersarang untuk constellations seasonal.
 				{
-					label: 'Styling',
+					label: 'Seasonal',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:153]: And same for this label.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 5]
```diff
@@ -139,15 +191,17 @@ Starlight dapat secara otomatis membuat grup di sidebar Anda berdasarkan direkto
...
 
-```js
+```js "label:" "autogenerate:"
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
-			// Hasilkan grup tautan secara otomatis untuk direktori 'guides'.
-			autogenerate: { directory: 'guides' },
+			label: 'Constellations',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:202]: This label can also be translated if it makes sense (and same goes for the example below).

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 6]
```diff
@@ -216,45 +269,44 @@ Sebuah grup yang dihasilkan secara otomatis termasuk halaman dengan frontmatter
...
-				{ label: 'Halaman lain', link: '#' },
+				{ label: 'Halaman lain', link: '' },
 			],
 		},
 	]}
 />
 
-:::note
-Konfigurasi frontmatter `sidebar` hanya digunakan untuk tautan yang dihasilkan secara otomatis dan akan diabaikan untuk tautan yang ditentukan secara manual.
+:::catatan
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:284]:
```suggestion
:::note
```

The aside type should not be translated.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 7]
```diff
@@ -216,45 +269,44 @@ Sebuah grup yang dihasilkan secara otomatis termasuk halaman dengan frontmatter
...
 
 Tautan, grup, dan grup yang dibuat secara otomatis juga dapat menyertakan properti `badge` untuk menampilkan _badge_ di samping labelnya.
 
-```js
+```js {9,16}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
+			label: 'Stars',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:296]: This label can be safely translated and updated in the preview below.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 8]
```diff
@@ -216,45 +269,44 @@ Sebuah grup yang dihasilkan secara otomatis termasuk halaman dengan frontmatter
...
 			items: [
-				// Tautan dengan badge "Baru".
+				// Tautan dengan badge "Supergiant".
 				{
-					label: 'Komponen',
-					link: '/guides/components/',
-					badge: 'Baru',
+					slug: 'stars/persei',
+					badge: 'Supergiant',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:301]: The badge can also be translated (and updated in the preview below too).

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 9]
```diff
@@ -216,45 +269,44 @@ Sebuah grup yang dihasilkan secara otomatis termasuk halaman dengan frontmatter
...
 			],
 		},
-		// Grup yang dibuat secara otomatis dengan badge "Ditinggalkan"
+		// Grup yang dibuat secara otomatis dengan badge "Outdated".
 		{
-			label: 'Referensi',
-			badge: 'Ditinggalkan',
-			autogenerate: { directory: 'reference' },
+			label: 'Moons',
+			badge: 'Outdated',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:308]: And same for this one.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 10]
```diff
@@ -265,54 +317,55 @@ Konfigurasi di atas akan menghasilkan sidebar berikut:
...
+			label: 'Stars',
 			items: [
-				// Sebuah tautan dengan badge "Eksperimental" berwarna kuning.
+				// Tautan dengan badge "Stub" berwarna kuning.
 				{
-					label: 'Komponen',
-					link: '/guides/components/',
-					badge: { text: 'Eksperimental', variant: 'caution' },
+					slug: 'stars/sirius',
+					badge: { text: 'Stub', variant: 'caution' },
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:368]: The badge and label can both be translated here and in the preview below.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 11]
```diff
@@ -343,16 +396,16 @@ Tautan juga dapat menyertakan properti `attrs` untuk menambahkan atribut HTML ku
 
 Pada contoh berikut, `attrs` digunakan untuk menambahkan atribut `target="_blank"`, sehingga tautan dibuka di tab baru, dan untuk menerapkan atribut `style` kustom untuk memberi style miring pada label tautan:
 
-```js
+```js {10}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
+			label: 'Resources',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:403]: This label can be safely translated here and the example below.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 12]
```diff
@@ -433,17 +570,14 @@ Menelusuri dokumentasi dalam Bahasa Portugis Brasil akan menghasilkan sidebar be
...
 Grup-grup tautan dapat dikembangkan secara default dengan mengatur properti `collapsed` menjadi `true`.
 
-```js
+```js {5-6}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
-			// Kembangkan grup secara default.
+			label: 'Constellations',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:577]: The label can be translated in this code block and in the example below.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 13]
```diff
@@ -454,26 +588,26 @@ Konfigurasi di atas akan menghasilkan sidebar berikut:
...
 [Grup-grup yang dihasilkan secara otomatis](#grup-grup-yang-dihasilkan-secara-otomatis) akan mengikuti nilai `collapsed` dari _parent group_ mereka:
 
-```js
+```js {5-6}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
-			// Kembangkan grup dan subgrup otomatis yang dihasilkan secara bawaan.
+			label: 'Constellations',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:607]: Same here for this label.

File: docs/src/content/docs/id/guides/sidebar.mdx [hunk 14]
```diff
@@ -484,33 +618,32 @@ Konfigurasi di atas akan menghasilkan sidebar berikut:
...
+Perilaku ini dapat ditimpa dengan mendefinisikan properti `autogenerate.collapsed`.
 
-```js
+```js {5-7} "collapsed: true"
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
-			// Jangan kembangkan grup "Panduan" tetapi kembangkan subgrup otomatisnya.
+			label: 'Constellations',
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/sidebar.mdx:642]: And also this one too.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few comments, mostly regarding some labels that can be safely translated. Other than that, the translation looks good to me.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 1 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx:117]: Ah i miss this one, thank you for the review. Will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 2 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: Ok, will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 3 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: I see, thank you for notice this. will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 4 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: will update soon

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 5 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: Ok, will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 6 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: i see, i will update it soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 7 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: Ok, will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 8 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: I see, i will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 9 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: Ok, will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 10 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: I see, thank you. will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 12 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: Ok, will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 14 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: Ok, will update soon

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 11 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: I see, will update soon.

File: docs/src/content/docs/id/guides/sidebar.mdx (hunk 13 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/sidebar.mdx]: will update soon.

**Review by HiDeoo — APPROVED** [review]
Thanks for the thorough updates, this is looks great to me now :rocket:

**Pull request merged**